File: BOJ_Python3/BOJ5543.py
# BOJ5543

hambuger = []
drink = []

for i in range(3):
    cost=int(input())
    hambuger.append(cost)
for i in range(2):
    cost=int(input())
    drink.append(cost)
print(min(hambuger)+min(drink)-50)

File: BOJ/BOJ5543.java
import java.util.*;
class BOJ5543{
    public static void main(String args[]){
        ArrayList<Integer> hambuger = new ArrayList<>();
        ArrayList<Integer> drink = new ArrayList<>();
        
        Scanner sc = new Scanner(System.in);
        
        for(int i=0; i<3; i++){
            hambuger.add(sc.nextInt());
        }
        
        for(int i=0; i<2; i++){
            drink.add(sc.nextInt());
        }
        
        System.out.println(Collections.min(hambuger)+Collections.min(drink)-50);
    }
}
